app/upgrades/types.go

```diff
@@ -22,3 +29,28 @@ type Upgrade struct {
 	// StoreUpgrades adds, renames and deletes KVStores in the state to prepare for a protocol upgrade.
 	StoreUpgrades storetypes.StoreUpgrades
 }
+
+type grantAuthorization struct {
```

Comment:
1. Why do we need to create a struct for this rather than using an existing protobuf type?
2. #PUC if we can't reuse something

Reply:
Removed it.